src/content/capture/canvas-pc/js/test.js

```diff
@@ -22,6 +22,9 @@ test('PeerConnection pc1 sample', function(t) {
       '/src/content/peerconnection/pc1/index.html')
   .then(function() {
     t.pass('page loaded');
+    // Override the trace function to ensure console logging works for
+    // webdriver.
+    seleniumHelpers.overrideTrace(driver);
```

Comment:
i would do this in buildDriver (which could reset/flush the output... hopefully)